docs/sources/operations/meta-monitoring/deploy.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Deploy Loki meta-monitoring
+menuTitle: Deploy Loki Meta Monitoring
+description: Describes how to deploy Meta Monitoring for Loki.
+weight: 100
+---
+
+# Deploy Loki meta-monitoring
+
+The primary method for collecting and monitoring a Loki cluster is to use the [Kubernetes Monitoring Helm](https://github.com/grafana/k8s-monitoring-helm/) chart. This chart provides a comprehensive monitoring solution for Kubernetes clusters and includes direct integrations for monitoring the full LGTM (Loki, Grafana, Tempo, and Mimir) stack. This tutorial will walk you through deploying the Kubernetes Monitoring Helm chart to monitor your Loki cluster.
```

Comment:
Word choice in the third sentence:  Is this a tutorial or a generic procedure?

"This procedure walks you through deploying the Kubernetes Monitoring Helm chart to monitor your Loki cluster."